### DISCUSSION TITLE
How to change screen size of labwc when opening in ubuntu?

### DISCUSSION THREAD

**Discussion by akshayr-hub** [labwc/labwc#2243] [Q&A]
How to change screen size of labwc when opening in ubuntu?

**Comment by johanmalm** [comment]
Not sure. My Ubuntu is too old to compile `wlroots` on. Can you not run `wlr-randr --output WL-1 --custom-mode <whatever>`?

**Reply by fungilife** [reply]
wlr-randr works just like xrandr ... flawlessly

Are you opening from X or from console as an independent session?
In either case wlr-randr will give the correct screen names connected

**Reply by akshayr-hub** [reply]
Thanks @johanmalm 
It worked
wlr-randr --output WL-1 --custom-mode "480x480"
